tests/filecheck/dialects/smt/invalid.mlir

```diff
@@ -3,27 +3,21 @@
 %arg1 = smt.constant true
 
 %and = smt.or %arg1
-// CHECK: operand at position 0 does not verify:
```

Comment:
this only passed because the `operand at position 0 does not verify:` was printed below so many times